src/jumps.jl

```diff
@@ -91,6 +108,59 @@ function VariableRateJump(rate, affect!;
                      save_positions, abstol, reltol)
 end
 
+"""
+$(TYPEDEF)
+
+Defines a jump process that pools together all of the jumps the user's model.
+In this process, the rate is a vector equation that computes the rates of each 
+jump process together:
+- [Tutorial](https://diffeq.sciml.ai/stable/tutorials/discrete_stochastic_example/)
+
+## Fields
+
+$(FIELDS)
+
+## Examples
+The `rate` function computes the rates for each jump process while `c` computes 
+total change matrix of all species in the system. Note that each columns of `regular_c`
+represent a different jump process. 
+
+```jldoctest; output = false
+using DiffEqJump
+using StableRNGs
+using LinearAlgebra
+rng = StableRNG(12345)
+
+function regular_rate(out, u, p, t)
+  out[1] = (0.1 / 1000.0) * u[1] * u[2]
+  out[2] = 0.01u[2]
+end
+
+function regular_c(du, u, p, t, counts, mark)
+    mul!(du, dc, counts)
+end
+
+dc = zeros(3, 2)
+rj = RegularJump(regular_rate, regular_c, 2)
+
+rj isa RegularJump
+
+# output
+
+true
+```
+
+Further, `RegularJump` problems require special solvers, such as
+`SimpleTauLeaping`. For example:
+
+```julia
+jumps = JumpSet(rj)
+prob = DiscreteProblem([999, 1, 0], (0.0, 250.0))
+jump_prob = JumpProblem(prob, Direct(), rj; rng = rng)
+sol = solve(jump_prob, SimpleTauLeaping(); dt = 1.0)
+```
```

Comment:
This example isn't complete

Reply:
Hey @ChrisRackauckas yeah, this is a good question to ask you. So the code above this sets up the `RegularJump`, and there is a doctest on creating that type of problem. In the code you tagged, this would actually solves that same problem. Question was, do you want to chew up CI time on running solves on the CI server? I was worried about adding a lot off extra time to the CI when building the docs. But if I should not worry about that, that is cool. But can you clarify what the right decision is. Thanks.

Reply:
don't worry about CI. If we need more CI we'll get more.

Reply:
Will do. Thanks for clarifying.